### PR TITLE
Adding Pomodoro-Timer made using react.js

### PR DESCRIPTION
**Pull request by VanshikaSabharwal**
### Issue Title: Pomodoro-Timer made using react.js

- **Name of the project** : Pomodor-Timer
- **Category:**  React.js
- **Name:**  Vanshika
- **GitHub ID:** [VanshikaSabharwal](https://github.com/VanshikaSabharwal)
- **Idenitfy yourself:**  GSSOC

Closes:  #55 

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made

I made a timer based on the pomodoro  technique which is 25minute work and 5 minute rest. So, after every 25 minutes , there will be a 5 minute rest. The design is easy to use and user-friendly as well as responsive. 

### Type of change ☑️

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### How Has This Been Tested? ⚙️

I have used ESLint for linting and testing to verify there is no bug.

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Screenshots 📷

Screenshot 2024-05-12 194609
Screenshot 2024-05-12 194542

### CODE REVIEW

File: React-JS-Projects/Basic/pomodoro-timer/screenshot.webp
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Basic/pomodoro-timer/screenshot.webp:1]: still ss is a empty file.

File: React-JS-Projects/README.md
Comment (Avdhesh-Varshney) [inline React-JS-Projects/README.md:1]: no correction same thing has done again.

File: database/react.json
Comment (Avdhesh-Varshney) [inline database/react.json:1]: change your vs code tab size to 4 OK.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@VanshikaSabharwal PR Not Approved 🎉 
Same mistake again repeated as of last pr.

File: database/react.json
Comment (VanshikaSabharwal) [reply database/react.json:1]: yes it's already 4

**Review by VanshikaSabharwal — COMMENTED** [review]
All files reviewed

**Review by Avdhesh-Varshney — COMMENTED** [review]
@VanshikaSabharwal leave this project, when you can't follow the guidelines properly.
This is too much to tell anyone again and again. This is not only waste of your time but my time also.

**Pull request closed without merging**